selfdrive/car/tests/routes.py

```diff
@@ -16,8 +16,10 @@
 
 # TODO: add routes for these cars
 non_tested_cars = [
+  FORD.BRONCO_SPORT_MK1,
```

Comment:
@adeebshihadeh We have some routes for the Bronco Sport, maybe we could use one of those?